packages/admin/src/commands/run.ts

```diff
@@ -0,0 +1,8 @@
+import { createCli } from "./create-cli"
```

Comment:
suggestion(non-blocking): Rename file to `index.ts` to be consistent with our other CLIs. 

Current file name could also be misleading in the sense that some think this exposes a command `run`